server/channels/store/searchlayer/layer.go

```diff
@@ -111,6 +111,28 @@ func (s *SearchStore) indexUser(rctx request.CTX, user *model.User) {
 	}
 }
 
+func (s *SearchStore) indexChannelsForTeam(rctx request.CTX, teamMember *model.TeamMember) {
+	channels, err := s.Channel().GetPublicChannelsForTeam(teamMember.TeamId, 0, 100)
+	if err != nil {
+		rctx.Logger().Warn("Encountered error while indexing public channels", mlog.String("team_member_id", teamMember.UserId), mlog.Err(err))
```

Comment:
Also string should be `Encountered error while getting all public channels for team` and we should log the team_id rather than the member_id. I don't believe the member_id has any role to play in the SQL query.